staging/proj.finassistant/manifest.tt

```diff
@@ -0,0 +1,139 @@
+// Copyright 2016-2020 The Board of Trustees of the Leland Stanford Junior University
```

Comment:
You're not a Stanford employee / RA. Are you donating copyright? Or do you want to retain it?